tmt/options.py

```diff
@@ -264,6 +264,23 @@ def option(
     )
 ]
 
+# TODO: Maybe this should be reversed.
+IMPORT_BEFORE_FILTER_OPTION: list[ClickOptionDecoratorType] = [
+    option(
+        '--import-before-filter',
```

Comment:
Chm, I'm looking at the help text, "Filtering by ``--filter`` and ``--condition`` are always applied after import and are not affected by this option." - would `--import-before-name-filter` make sense? `before-filter` combined with "yeah, but `--filter` and `--condition` are not affected, it's really just one kind of filtering" behavior feels weird :/